templates/kubernetes/config.yml

```diff
@@ -1,5 +1,5 @@
 name: Kubernetes
 description: |
   Kubernetes service
-version: v1.2.4-rancher6
+version: v1.2.4-rancher7
```

Comment:
during my initial review I thought we'd include @prachidamle changes for k8s, but we are postponing it to 1.2. So no new k8s image, and as the template version correlates to k8s image id (currently v1.2.4-rancher6), we should perhaps use v1.2.4-rancher6-1 or v1.2.4-rancher6.1 as per https://github.com/rancher/rancher-catalog/pull/163

@ibuildthecloud @vincent99 which format should we go for?